src/org/jitsi/impl/neomedia/rtp/translator/SimulcastController.java

```diff
@@ -260,6 +260,13 @@ public boolean accept(byte[] buf, int off, int len)
      */
     public void setTargetIndex(int newTargetIdx)
     {
+        int oldTargetIdx = bitstreamController.getTargetIndex();
+        if (oldTargetIdx == newTargetIdx)
+        {
+            // Do not spam with FIRs.
```

Comment:
this comment is a bit confusing.  i think i understand that if the new index is the same we don't need to do anything else, so maybe this comment should be something more broad like `No work necessary`?